services/softlayer_product_package_test.go

```diff
@@ -72,4 +72,90 @@ var _ = Describe("SoftLayer_Product_Package", func() {
 			Expect(itemPrices[0].Item.Id).To(Equal(456))
 		})
 	})
+
+	Context("#GetItems", func() {
+		BeforeEach(func() {
+			fakeClient.DoRawHttpRequestResponse, err = testhelpers.ReadJsonTestFixtures("services", "SoftLayer_Product_Package_getItems.json")
+			Expect(err).ToNot(HaveOccurred())
+		})
+
+		It("returns an array of datatypes.SoftLayer_Product_Item", func() {
+			productItems, err := productPackageService.GetItems(222)
+			Expect(err).ToNot(HaveOccurred())
+			Expect(len(productItems)).To(Equal(2))
+			Expect(productItems[0].Id).To(Equal(123))
+			Expect(productItems[0].Prices[0].Id).To(Equal(456))
+		})
+	})
+
+	Context("#GetItemsByType", func() {
+
+		BeforeEach(func() {
+			response, err := testhelpers.ReadJsonTestFixtures("services", "SoftLayer_Product_Package_getAllObjects_virtual_server.json")
+			fakeClient.DoRawHttpRequestResponses = append(fakeClient.DoRawHttpRequestResponses, response)
+
+			Expect(err).ToNot(HaveOccurred())
+		})
+
+		It("returns an array of datatypes.SoftLayer_Product_Item", func() {
+			response, err := testhelpers.ReadJsonTestFixtures("services", "SoftLayer_Product_Package_getItems.json")
+			fakeClient.DoRawHttpRequestResponses = append(fakeClient.DoRawHttpRequestResponses, response)
+
+			productItems, err := productPackageService.GetItemsByType("VIRTUAL_SERVER_INSTANCE")
+			Expect(err).ToNot(HaveOccurred())
+			Expect(len(productItems)).To(Equal(2))
+			Expect(productItems[0].Id).To(Equal(123))
+			Expect(productItems[0].Prices[0].Id).To(Equal(456))
+		})
+	})
+
+	Context("#GetPackagesByType", func() {
+		BeforeEach(func() {
+			fakeClient.DoRawHttpRequestResponse, err = testhelpers.ReadJsonTestFixtures("services", "SoftLayer_Product_Package_getAllObjects_virtual_server.json")
+			Expect(err).ToNot(HaveOccurred())
+		})
+
+		It("returns an array of datatypes.Softlayer_Product_Package", func() {
+			productPackages, err := productPackageService.GetPackagesByType("VIRTUAL_SERVER_INSTANCE")
+			Expect(err).ToNot(HaveOccurred())
+			Expect(len(productPackages)).To(Equal(4))
+			Expect(productPackages[0].Id).To(Equal(200))
+			Expect(productPackages[0].Name).To(Equal("Cloud Server 1"))
+		})
+
+		It("outlet packages are skipped", func() {
```

Comment:
So the `It(...)` needs to read like English. "It outlet packages are skipped" does not read well. Please change to something like: `It("skips packaged marked OUTLET")` or similar.